lib/ap/plan9/signal.c

```diff
@@ -64,7 +64,10 @@ void
 	_sighdlr[sig] = func;
 	return oldf;
 }
-
+/*
+ * Wrong, for learning purposes.
+ * Real is in lib/ap/amd64 directory
+ *
 int
 sigsetjmp(sigjmp_buf buf, int savemask)
```

Comment:
get this out of here. It's going to confuse somebody in a big way in the future.